src/components/hero/Hero.jsx

```diff
@@ -0,0 +1,28 @@
+import { banner } from "assets/index";
+import "./hero.css";
+import { Link } from "react-router-dom";
+
+function Hero() {
+  return (
+    <main className="main">
+      <div className="banner-text">
+        <h1 className="fs-l">Meet your Modern</h1>
+        <h1 className="fs-l secondary-text">Note Keeping App</h1>
```

Comment:
Here instead of creating two separate h1 you can use just one and wrap the "Note keeping App" in span/div and give it a class of secondary-text.

Reply:
Changed it